content/en/docs/reference/using-api/server-side-apply.md

```diff
@@ -578,8 +578,7 @@ Apply can send partially specified objects as YAML as the body of a `PATCH` requ
 to the URI of a resource.  When applying a configuration, you should always include all the
 fields that are important to the outcome (such as a desired state) that you want to define.
 
-All JSON messages are valid YAML. Some clients specify Server-Side Apply requests using YAML
-request bodies that are also valid JSON.
+All JSON messages are valid YAML. Therefore, in addition to using YAML request bodies for Server-Side Apply requests, you can also use JSON request bodies, as they are also valid YAML.
 
```

Comment:
```suggestion
In either case, use the media type `application/apply-patch+yaml` for the HTTP request.

```

Reply:
I recommend the extra sentence to avoid the risk of people inferring that they can use `application/json` or `application/apply-patch+json` as the media type.